src/index.html

```diff
@@ -6,7 +6,43 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="
+      calendar
+      page__calendar
+      calendar--month-lenght--30
+    ">
```

Comment:
```suggestion
    <div class="
      calendar
      page__calendar
    ">
```

More than one block per tag, run linter before commit

Checklist №1